client/consensus/slots/src/lib.rs

```diff
@@ -77,6 +79,7 @@ pub trait SlotWorker<B: BlockT, Proof> {
 	async fn on_slot(
 		&mut self,
 		slot_info: SlotInfo<B>,
+		justification_sync_link: &mut dyn JustificationSyncLink<B>,
```

Comment:
Please move this to the `SimpleSlotWorker` trait as it is done with the other values we need in the `on_slot` implementation.,

Reply:
Yeah I was a bit lazy, also the reason I'm using a trait object here 😊. Will update.